contracts/FlowIDTableStaking.cdc

```diff
@@ -1041,8 +1019,17 @@ pub contract FlowIDTableStaking {
     }
 
     init(_ epochTokenPayout: UFix64, _ rewardCut: UFix64) {
+        self.account.save(true, to: /storage/stakingEnabled)
+
         self.nodes <- {}
 
+        let stakingKeyClaimed: {String: Bool} = {}
+        self.account.save(stakingKeyClaimed, to: /storage/stakingKeysClaimed)
+        let networkingKeyClaimed: {String: Bool} = {}
+        self.account.save(networkingKeyClaimed, to: /storage/networkingKeysClaimed)
+        let networkingAddressClaimed: {String: Bool} = {}
+        self.account.save(networkingAddressClaimed, to: /storage/networkingAddressesClaimed)
```

Comment:
Maybe simplify to

```suggestion
        self.account.save({}, to: /storage/stakingKeysClaimed)
        self.account.save({}, to: /storage/networkingKeysClaimed)
        self.account.save({}, to: /storage/networkingAddressesClaimed)
```

Reply:
Should that be possible? When I try to do that, I get this error:
```
expected `Storable`, got `{Never: Never}`
```
I think I still need to do it the way I have it because I need to specify the type. Is there a better way to do it?